burglebros.game.php

```diff
@@ -1919,6 +1919,22 @@ function handleEventEffect($player_id, $card) {
             
             $this->performGuardMovementEffects($guard_token, $patrol_token['location_arg']);
             $this->nextPatrol($floor);
+        } elseif ($type == 'squeak') {
+            // Guard moves 1 tile towards the closest player
+            $tile = $this->getPlayerTile($player_id);
+            $floor = $tile['location'][5];
+            $guard_token = array_values($this->tokens->getCardsOfType('guard', $floor))[0];
+            $guard_tile = $this->tiles->getCard($guard_token['location_arg']);
+            $shortest_path = null;
+
+            $players = self::loadPlayersBasicInfos();
+            foreach ($players as $player_id => $player) {
+                $player_token = $this->getPlayerToken($player_id);
+                $player_tile = $this->getPlayerTile($player_id, $player_token);
+                $path = $this->findShortestPathClockwise($floor, $guard_tile['location_arg'], $player_tile['location_arg']);
```

Comment:
There might be a tie, in which case the player gets to pick which tile the guard moves to.

Reply:
ok, i'll add this to the TODO in the Google Docs (I recall there is another case where player has to choose guard's path, i'll implement both)